shared/middleware/helpers/storage_apps.rb

```diff
@@ -74,7 +74,18 @@ def publish(channel_id, type)
     update_count = @table.where(id: id).exclude(state: 'deleted').update(row)
     raise NotFound, "channel `#{channel_id}` not found" if update_count == 0
 
-    row[:published_at]
+    project = @table.where(id: id).first
+    project_value = project[:value] ? JSON.parse(project[:value]) : {}
```

Comment:
good point -- `project[:value]` should never be nil. If it is, we'd like honeybadger to let us know.